src/functionHandlers/notarisePdt/handler.ts

```diff
@@ -24,6 +28,7 @@ export interface NotarisationResult {
   notarisedDocument: WrappedDocument<HealthCertDocument>;
   ttl: number;
   url: string;
+  qrCode: Buffer;
```

Comment:
Oh, why the additional return value for the qr code?

Reply:
I was following the implementation of `uploadDocument()` in api-notarise-formsg which returned both `ttl` and `qrCode`.

Let me remove the return of `qrCode` in the `uploadDocument()` function and generate it inline when triggering the notification.

Reply:
Thanks. The QR code isn't needed in this context because it only needs to be generated for triggering the notification to SPm